app/models/step_by_step_page.rb

```diff
@@ -15,6 +15,7 @@ class StepByStepPage < ApplicationRecord
   validates :slug, format: { with: /\A([a-z0-9]+-)*[a-z0-9]+\z/ }, uniqueness: true
   validates :slug, slug: true, on: :create
   validates :status, inclusion: { in: STATUSES }, presence: true
+  validates :status, status: true
```

Comment:
What about renaming the validator as it took me quite a while to realise that the second `:status` was the name of the validator?

Reply:
I've renamed the validator to `StatusPrerequisiteValidator`.

Reply:
Updated and fixed-up